horovod/common/controller.cc

```diff
@@ -628,53 +619,43 @@ void Controller::CoordinateCacheAndState(CacheCoordinator& cache_coordinator) {
                                (Request::RequestType)response.response_type());
     }
 
-    // End negotation phase for synced cache hit set entries.
+    // End negotiation phase for synced cache hit set entries.
     for (auto bit : cache_coordinator.cache_hits()) {
       auto& response = response_cache_.peek_response(bit);
       timeline_.NegotiateEnd(response.tensor_names()[0]);
     }
   }
 }
 
-ResponseList Controller::FuseResponses(std::deque<Response>& responses,
-                                       bool joined) {
+ResponseList Controller::FuseResponses(std::deque<Response>& responses) {
```

Comment:
This needs to be updated in the header as well.

Reply:
Forgot to add to the commit, fixed.